core/src/search_stores/indices/nodes_1.settings.local.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
why is it named nodes_1 ?

We should be likely more precise in our index name this is a core index this index is for data sources and within data sources it's for nodes so at least core_data_sources_nodes

Reply:
core prefixing => it's what is already done, along with region. `us-central-1.core.nodes_1` will be accessed as `core.nodes` via the elasticsearch aliasing system

name of this particular index => imo it's not really data sources but nodes that we index here, furthermore an index does not map to a table so the need to name it like the table is limited and could even be a little misleading

I was personally in favor of aligning to `tables` table (that doesn't have the data_sources prefix)

Although it's not that important for me so I'm fine to rename `nodes` into `data_sources_nodes` if you deem it necessary :+1:

Reply:
well the `nodes` SQL table should be named `data_sources_nodes` if you asked me :) #consistency

Reply:
you mean the `tables` sql table ? (nodes is already named like that)